app/app.ts

```diff
@@ -108,6 +107,9 @@ function setupIpc(localSettings: LocalSettings, scrSettings: ScrSettings) {
     })
     .on('settingsLocalMerge', (event, settings) => {
       // This will trigger a change if things changed, which will then emit a LOCAL_SETTINGS_CHANGED
+      app.setLoginItemSettings({
```

Comment:
I'm not sure if there's a better place to do this, It'd be most natural code organization wise to put this at the top of the file with the other electron settings but i'm not sure that's possible since it relies on an ipc call

Reply:
A better place for this would probably be in the `change` event handler that the local settings emit (we're listening to it in the few lines below). That event is emitted only when something actually changes in the settings file, and is emitted if someone manually changes their settings file.

Also, I'm not sure about the performance implications of running this each time _any_ setting changes. I assume it's simply writing something in the registry (at least on Windows machines), so it shouldn't be too bad. Alternatively, you could call the [getLoginItemSettings()](https://www.electronjs.org/docs/api/app#appgetloginitemsettingsoptions-macos-windows) and check if this setting actually changed before trying to set it.